extensions-contrib/prometheus-emitter/src/main/java/org/apache/druid/emitter/prometheus/PrometheusEmitterConfig.java

```diff
@@ -53,6 +53,10 @@
   @Nullable
   private final String pushGatewayAddress;
 
+  @JsonProperty
+  @Nullable
+  private final Integer pushRateSec;
```

Comment:
Better to name it as `flushPeriod` since it's the same property name in other emitter extensions.

Reply:
ok, looks good